packages/ilios-common/addon/classes/events-base.js

```diff
@@ -115,6 +115,27 @@ export default class EventsBase extends Service {
     obj.postrequisites = sortBy(obj.postrequisites, 'startDate');
     obj.postrequisites = sortBy(obj.postrequisites, 'name');
     obj.isUserEvent = isUserEvent;
+
+    // The start and end date of the event, for display purposes on the calendar. See comment block below.
+    obj.calendarStartDate = obj.startDate;
```

Comment:
I like this rename, in the followup to this when we build events as classes we should also drop outside access to startDate and endDate if possible.